### ISSUE TITLE
Focusable image with empty alt

### ISSUE THREAD

**Issue by giacomo-petri** [auto-wcag/auto-wcag#2277]
## Accessibility support

> Images can have their role set to presentation through an empty alt attribute. [Presentational Roles Conflict Resolution](https://www.w3.org/TR/wai-aria-1.1/#conflict_resolution_presentation_none) does not specify what to do if such an image is [focusable](https://www.w3.org/WAI/standards-guidelines/act/rules/23a2a8/#focusable) (it only specifies what to do in case of explicit role="none" or role="presentation"). Some browsers expose these images and some don’t. Thus, this rule may fail for technologies that expose these without creating an accessibility issue for users of other technologies.

## ACT Rules

- [Image has non-empty accessible name - 23a2a8](https://www.w3.org/WAI/standards-guidelines/act/rules/23a2a8/)

### Test case 1

Image element with empty acc name and non-negative tabindex attribute value
```html
<img src="some_src" alt="" tabindex="0">
```

[Test cases](https://codepen.io/giacomo-usablenet/full/xbKoBvM)

#### Test instructions

- Just run the test and check the element role

### External links

- (I'll open a WPT test for this one)
- WAI-ARIA: missing ARIA specs, opened https://github.com/w3c/aria/issues/2415

**Comment by giacomo-petri** [comment]
## Test results

### Test case 1

I've run them locally and all browsers are exposing the element as a role image

```md
## Test case 1

- Date: 29 Jan 2025
- Operating system: not relevant
- Browser: Safari, Chrome, Firefox
- Assistive technology: not relevant
- Test result: Passed
```
